### DISCUSSION TITLE
Time to Target vs Ready

### DISCUSSION THREAD

**Discussion by pelican1997** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#489] [General]
I'm sorry if this is stated somewhere, but I couldn't find it. What's the difference between "Time to Target" vs "Time to Ready?"

**Comment by visualapproach** [comment]
Time to target is obsolete and is not used in latest code. It's a naive way of calculating by extrapolating the temperature curve linearly. Time to ready is considering the ambient temperature and would be more exact if calibrated. Also the latter doesn't care if the heater is on or not, it calculates the time it will take if you were to turn on the heater. It is also used to be able to set a time when you would like the tub to be ready. For the moment there is a logic error in the calibration function but a fix is on it's way already.

**Comment by pelican1997** [comment]
Makes sense! Thanks!
